lib/tasks/db.rake

```diff
@@ -1,4 +1,20 @@
 namespace :db do
+  namespace :connection do
+    desc "Check ActiveRecord connection to database"
+    task check_connection: :environment do
+      ActiveRecord::Base.establish_connection # Establishes connection
+      ActiveRecord::Base.connection # Calls connection object
+      # output appropriate response depending on whether ActiveRecord can establish connection or not
+      if ActiveRecord::Base.connected?
+        print "ActiveRecord was able to establish connection with DB"
+      else
+        print "ActiveRecord unable to establish connection with DB"
+      end
+    rescue PG::ConnectionBad, ActiveRecord::ConnectionNotEstablished
+      print "ActiveRecord unable to establish connection with DB"
```

Comment:
Since it's the last step, it won't print out twice - if any of these exceptions happen, as soon as they happen it will "exit" and do the rescue print

If there's an issue that means the db couldn't connect that is unrelated to these exceptions, it will print via the if/else statement. The reason I have the exception rescues is because these errors can happen before the connected? check when we are attempting to establish the connection